docs/build/identity.rs/1.3/docs/references/specifications/revocation-timeframe-2024.mdx

```diff
@@ -0,0 +1,149 @@
+:::
+
+## Data Model
+
+### Revocation Timeframe Status
+
+For an issuer to enable verifiers to check the status of a verifiable credential, the [`credentialStatus`](https://www.w3.org/TR/vc-data-model/#status) property MUST be specified with the following properties:
+
+| Property                | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| :---------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `id`                    | The constraints on the `id` property are listed in the [Verifiable Credentials Data Model specification](https://www.w3.org/TR/vc-data-model/). The `id` MUST be a [DID URL](https://www.w3.org/TR/did-core/#did-url-syntax) that is the URL to a [Revocation Bitmap Service](#revocation-bitmap-service) in the DID Document of the issuer. |
+| `type`                  | The `type` property MUST be `"RevocationBitmap2022"`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| `revocationBitmapIndex` | The `revocationBitmapIndex` property MUST be an unsigned, 32-bit integer expressed as a string. This is the index of the credential in the issuers revocation bitmap. Each index SHOULD be unique among all credentials linking to the same [Revocation Bitmap Service](#revocation-bitmap-service).                                                                                                                                                                                                                                                                                                                          |
```

Comment:
I would add a sentence at the end like this: 

""This value is _not disclosed_ to the verifier by default"."